cmd/nodeadm/init/init.go

```diff
@@ -72,6 +74,10 @@ func (c *initCmd) Run(log *zap.Logger, opts *cli.GlobalOptions) error {
 		} else if err != nil {
 			return err
 		}
+
+		if err := containerd.ValidateSystemdUnitFile(); err != nil {
+			return fmt.Errorf("please install systemd unit file for containerd: %v", err)
```

Comment:
nit:
```suggestion
			return fmt.Errorf("a systemd unit file for containerd is required to init the node: %w", err)
```

Reply:
I preserved the exact wording from before, but happy to update it.